lib/fuse_lowlevel.c

```diff
@@ -3137,6 +3183,18 @@ int fuse_session_receive_buf_int(struct fuse_session *se, struct fuse_buf *buf,
 	return res;
 }
 
+int fuse_session_receive_buf(struct fuse_session *se, struct fuse_buf *buf)
+{
+	return _fuse_session_receive_buf(se, buf, NULL, false);
+}
+
+/* libfuse internal handler */
+int fuse_session_receive_buf_int(struct fuse_session *se, struct fuse_buf *buf,
```

Comment:
`int` as in `internal`? perhaps verbose but I would suggest the full word for clarity.